numpy/random/tests/test_generator_mt19937.py

```diff
@@ -2558,13 +2555,15 @@ def gen_random(state, out):
 
 # See Issue #4263
 class TestSingleEltArrayInput:
-    def setup_method(self):
-        self.argOne = np.array([2])
-        self.argTwo = np.array([3])
-        self.argThree = np.array([4])
-        self.tgtShape = (1,)
+    def _create_arrays(self):
+        argOne = np.array([2])
+        argTwo = np.array([3])
+        argThree = np.array([4])
+        tgtShape = (1,)
+        return argOne, argTwo, argThree, tgtShape
```

Comment:
IMO in this case binding the variable names just makes this harder to read, since the one-liner fits nicely on one line.

```suggestion
        return np.array([2]), np.array([3]), np.array([4]), (1,)
```